leasing/fixtures/vat.json

```diff
@@ -23,6 +23,15 @@
     "fields": {
       "percent": 24,
       "start_date": "2013-01-01",
+      "end_date": "2024-12-31"
+    }
+  },
+  {
+    "model": "leasing.Vat",
+    "pk": 4,
+    "fields": {
+      "percent": 25.5,
+      "start_date": "2025-01-01",
```

Comment:
Should we actually put an earlier dates at first in order to do testing with Laske? Then, when it is tested, it could be changed back to what it is supposed to be, before production release?

Reply:
Could you go a bit more in detail on what you mean, which order would you like to change?
Is the VAT change or its date fully confirmed yet? If we are not sure, it might be a better idea to add the data later.